### ISSUE TITLE
improved issue template

### ISSUE THREAD

**Issue by foss-** [friendica/friendica#9235]
The current template has some problems. Below is a proposal for a better template. Markdown code is:

```
<!--
Please fill out this template with all the information you have. The more info you can provide the easier it will be to help you out or fix the problem you are seeing. For trouble with the UI don’t forget to add a screenshot or two. Please do your best!

Please note that this tracker is only for bugs and feature requests. Please try these
locations if you have a question or comment:

  https://forum.friendi.ca/profile/helpers
  Reach this forum by adding `!helpers@forum.friendi.ca ` to your friendica post

Lastly, be sure to preview your issue before saving. Thanks!
-->

- [ ] I have searched open and closed issues for duplicates

<!--
  You can search all issues here
  https://github.com/friendica/friendica/issues?utf8=%E2%9C%93&q=is%3Aissue
  Replace [ ] with [X] once you've searched
-->

### Bug Description

<!-- Give an overall summary of the issue. -->

### Steps to Reproduce

<!-- Using bullet points, list the steps that reproduce the bug. -->

1.  step one
2.  step two
3.  step three

Actual Result:

<!-- Describe the details of the buggy behaviour. -->

Expected Result:

<!-- Describe in detail what the correct behavior should be. -->

### Screenshots

<!-- How to take screenshots on all OSes: https://www.take-a-screenshot.org/
You can drag and drop images into this text box. -->

### Platform Info

Friendica Version:

<!-- You can see Friendica’s version number at Help -> About this site -> Site/Friendica Version -->

Friendica Source: 

PHP version:

SQL version:
```

### Once posted looks like this:

- [ ] I have searched open and closed issues for duplicates

### Steps to Reproduce

1.  step one
2.  step two
3.  step three

Actual Result:

Expected Result:

### Platform Info

Friendica Version:

Friendica Source: 

PHP version:

SQL version:

**Comment by MrPetovan** [comment]
> ```
> Please note that this tracker is only for bugs and feature requests. Please try these
> locations if you have a question or comment:
>
>  https://forum.friendi.ca/profile/helpers
>  Reach this forum by adding `!helpers@forum.friendi.ca ` to your friendica post
> ```

I disagree with this, we have an issue label dedicated to user support so questions are welcome here. Comments, not as much, so the wording stands but without the `question` word.

I would also put the Expected Result title before the Actual Result title, and it sounds redundant with "Bug description".

**Comment by foss-** [comment]
My subjective preference is to keep bugs / features and support separate. Clearly if friendica devs like to use GitHub for support requests as well please alter that part as required.

Regarding using `Bug Description` or not: Often times it is better to request more information. And Bug description could be a very brief description, sometimes identical to title or could also be removed by the user. When personally filing bugs I often find myself debugging a problem or investigating further while filling a good form. That then results in better reports for the project. So asking about a description won't hurt, even if partly redundant.

**Comment by MrPetovan** [comment]
I feel like the steps to reproduce, the expected result and the actual result are a good starting point. We could ask for a vague "Context" in the form but since it usually depends on the problem, I do not have a problem with asking pointed questions about it.

**Comment by MrPetovan** [comment]
Closed by #9275
